### ISSUE TITLE
How does this compare to others?

### ISSUE THREAD

**Issue by warvstar** [k8gb-io/k8gb#689]
Okay so I'm just getting involved in k8s now, and load balancers.

I currently use AWS load balancers attached to AWS Global Accelerator (anycast), as I was primarily using ec2 servers directly.

Now I'm looking into using containers and scaling across cloud providers. Can I still use AWS for this? Should I use k8s? What about Github's Load Balancer director?

I know I just said I'm interested in multicloud, but what if I didn't want to do multicloud, would there be a reason to use anything other than my current setup for load balancing?

My use case is this is: I have some immutable data stored in memory, I want users from all over the world to be able to connect to the closest (or fastest?) server near them, so they can download these assets as quickly as possible (generally via websockets, as downloading 1000s of assets via GET requests has overhead that I measured to be unacceptable.)

Basically like a CDN; the only reason I'm not using a CDN is because from my tests, even if they have edge locations in my city, it can be quite a bit slower downloading via GET requests and also because I'd like more control over my cache. On my machines I can keep recently accessed data in memory for as long as I'd like.

Also, if anyone knows of a cost and performance equivalent or better using serverless, then I'd be interested in knowing about that as well.

Btw, I'm willing to pay for help and advice here, thanks all!

**Comment by ytsarev** [comment]
@warvstar hope it answers your questions, closing, feel free to reopen in case you want to continue the conversation
